CVX/manual_work/cvx_refresh.sql

```diff
@@ -0,0 +1,147 @@
+--Backup concept_relationship_manual table and concept_manual table.
```

Comment:
@MariaRohozhkina Manual work is not described in the readme file

Reply:
Done